fenix/app/src/test/java/org/mozilla/fenix/shopping/ui/StarRatingKtTest.kt

```diff
@@ -0,0 +1,27 @@
+package org.mozilla.fenix.shopping.ui
+
+import junit.framework.TestCase.assertEquals
+import org.junit.Test
+
+class StarRatingKtTest {
+
+    @Test
+    fun `GIVEN a float THEN removeDecimalZero should remove zero if present`() {
```

Comment:
We generally try to avoid referencing the function name inside the test name. That way the test seems more agnostic to what it's specifically testing, and it's less refactor work if that function ever changes.

```suggestion
    fun `GIVEN a float with a decimal THEN remove the zero if present`() {
```

Reply:
fair point!